scripts/in_container/_in_container_utils.sh

```diff
@@ -259,11 +259,15 @@ function install_released_airflow_version() {
         export SLUGIFY_USES_TEXT_UNIDECODE=yes
     fi
     rm -rf "${AIRFLOW_SOURCES}"/*.egg-info
-    INSTALLS=("apache-airflow==${1}" "werkzeug<1.0.0")
-    pip install --upgrade "${INSTALLS[@]}"
+    if [[ ${INSTALL_AIRFLOW_VERSION} == "wheel" ]]; then
+        pip install /dist/apache_airflow-*.whl
+    else
+        INSTALLS=("apache-airflow==${1}")
+        pip install --upgrade "${INSTALLS[@]}"
```

Comment:
@ashb Removed werkzeug. It was needed before we got constraints and I checked that everything installs just fine down to 1.10.2 version.